src/conf_mode/service_pppoe-server.py

```diff
@@ -73,7 +73,8 @@ def get_config(config=None):
     # https://phabricator.accel-ppp.org/T3
     conditions = [is_node_changed(conf, base + ['client-ip-pool']),
                   is_node_changed(conf, base + ['client-ipv6-pool']),
-                  is_node_changed(conf, base + ['interface'])]
+                  is_node_changed(conf, base + ['interface']),
+                  is_node_changed(conf, base + ['authentication','radius','dynamic-author'])]
```

Comment:
Try to change authentication method from local to RADIUS it also required restart the service